docs/fleet/provision.md

```diff
@@ -1,278 +1,186 @@
+     This is the JSON object which contains your machine part secret key and cloud app address, which your machine's `viam-server` instance needs to connect to the Viam app.
 
-Viam Agent only sends messages when your machine is online and connected to the internet.
-If your machine is offline, log messages are queued, and are sent to the Viam app once your machine reconnects to the internet.
+     To copy a machine's `viam-server` app configuration:
 
-These log messages include when `viam-server` is stopped and started, the status of agent subsystems, and any errors or warnings encountered during operation.
+     - Navigate to your machine's page on [the Viam app](https://app.viam.com) and select the **CONFIGURE** tab.
+     - Select the part status dropdown to the right of your machine's name on the top of the page:
+       {{<imgproc src="/build/micro-rdk/part-dropdown.png" resize="x600" style="max-width: 500px" declaredimensions=true alt="The part status dropdown of an offline machine.">}}
+     - Click the copy icon underneath **Viam server configuration** to copy the `viam-server` app JSON configuration.
+     - Paste the `viam-server` app config when prompted.
 
-## Provision a new machine
+1. The machine will then disable the hotspot network and attempt to connected using the provided network information.
+   If `viam-agent` cannot establish a connection using the provided network information, the machine will create the hotspot again and continue going through steps (2-5) until a connection is successfully established.
+1. If the connection is successful, `viam-agent` installs `viam-server`.
 
-With the Viam Agent installed, your machine will either connect to a local WiFi network or will create its own WiFi hotspot, depending on your configuration.
+   - `agent-provisioning` will use the provided network if it can connect, even if that network does not have internet access.
+     Note that any features that require internet access will not function if the connected WiFi network is not connected to the internet.
+     If you want `agent-provisioning` to require that a WiFi network be connected to the internet in order to connect to it, enable roaming mode **and** do not provide a primary network.
 
-- If you include a `viam-server` configuration file on your machine, located at <file>/etc/viam.json</file>, which includes a WiFi network and password to connect to, the Viam Agent will connect to the network automatically when in range.
-- If you did not include this file, or the configured WiFi network is not available when your machine comes online, the Viam Agent will create its own WiFi hotspot and request WiFi credentials:
+1. `viam-agent` then starts `viam-server` with a provided configuration and the machine comes **live**.
+   {{% /tab %}}
 
-  ![Diagram of the flow of provisioning a new machine with the Viam agent.](/platform/provision/style-provisioning-diagram.png)
+{{% tab name="With pre-configured network (Roaming mode)" %}}
```

Comment:
I'm not sure this tab makes sense at all. Adding (additional) networks via the agent-config (in the cloud) only applies AFTER provisioning is first completed. The config is stored in the cloud (App) so isn't downloaded to the machine until after the machine first connects anyway.

Reply:
If a manufacturer wants to pre-install network configurations on an unprovisioned device, they should do so via the OS mechanisms directly. E.g. Add connections directly in NetworkManager.